docs/metasploit-framework.wiki/Setting-Up-A-AD-CS-Target.md

```diff
@@ -0,0 +1,61 @@
+
+# How To Set Up An ESC Target
+## Installing AD CS
+- [ ] Install AD CS on either a new or existing domain controller
+    - [ ] Open the Server Manager
+    - [ ] Select Add roles and features
+    - [ ] Select "Active Directory Certificate Services" under the "Server Roles" section
+    - [ ] When prompted add all of the features and management tools
+    - [ ] On the AD CS "Role Services" tab, leave the default selection of only "Certificate Authority"
+    - [ ] Completion the installation and reboot the server
+    - [ ] Reopen the Server Manager
+    - [ ] Go to the AD CS tab and where it says "Configuration Required", hit "More" then "Configure Active Directory Certificate..."
+    - [ ] Select "Certificate Authority" in the Role Services tab
+    - [ ] Keep all of the default settings, noting the value of the "Common name for this CA" on the "CA Name" tab (this value corresponds to the `CA` datastore option)
```

Comment:
I'd like to add this additional detail:

```suggestion
    - [ ] Select "Certificate Authority" in the Role Services tab
    - [ ] Select "Enterprise CA" in the "Setup Type" tab (the user must be a Domain Administrator for this option to be available)
    - [ ] Keep all of the default settings, noting the value of the "Common name for this CA" on the "CA Name" tab (this value corresponds to the `CA` datastore option)
```

Reply:
Added suggestion in https://github.com/rapid7/metasploit-framework/pull/17399/commits/239bc42f4c4228a21dd726e08fa3273545112f7f